tests/test_goapidiff.py

```diff
@@ -0,0 +1,88 @@
+import json
+import os
+
+import pytest
+
+from system.plugins.goapidiff.analyzer import GoApiDiff
+
+
+TESTDATA_DIR = 'testdata'
+
+PATH_PAIRS_IDENTICAL_COMMITS = [
+    [
+        'github.com/coreos/etcd/b4bddf685b26b4aa70e939445044bdeac822d042',
```

Comment:
Ahh. I see know. All tests rely on the directory hierarchy created by the ``prepare`` function.